pkgs/development/python-modules/cardimpose/default.nix

```
@@ -0,0 +1,37 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+  pymupdf,
+}:
+buildPythonPackage rec {
+  pname = "cardimpose";
+  version = "0.2.1";
+  pyproject = true;
```

```suggestion
  pyproject = false;
```
Nitpick: While this does work, *technically* the project doesn't support PEP517/have a pyproject.toml

---

Based on https://nixos.org/manual/nixpkgs/stable/#buildpythonpackage-parameters I had the impression `pyproject = true` should be set on all Python packages even if they're only using setup.py. Given that, I'd be inclined to leave as-is, although I'm far from an expert on either Python or Nix packaging, so if it's going to block merging this I'm happy to make the change.

---

I've always felt that chapter is a bit vague since "all other formats are deprecated" could be referring to them being deprecated by Python or Nixpkgs itself...but AFAIK there is nothing else in Nixpkgs mentioning `setuptoolsBuildHook`, etc are deprecated. It also goes on to specifically mention the use of hooks right after `setuptools` compat, which I've always taken as advice that it's an alternative 🤷 

> Note that the pyproject format falls back to using setuptools, so you can use pyproject = true even if the package only has a setup.py. When set to false, you can use the existing [hooks](https://nixos.org/manual/nixpkgs/stable/#setup-hooks) or provide your own logic to build the package.

Since this is sorta unclear (and a nitpick on my part honestly 😆), it's not going to be a blocker ofc. Just a suggestion ;)